bindings/nodejs/lib/client/client.ts

```diff
@@ -345,7 +346,7 @@ export class Client {
     /**
      * Post block as raw bytes, returns the block ID.
      */
-    async postBlockRaw(block: Block): Promise<BlockId> {
+    async postBlockRaw(block: BlockWrapper): Promise<BlockId> {
```

Comment:
Can we keep the old name?

Reply:
Yea we discussed it on slack a bit, lets discuss it on next sprint planning and leave for now